src/index.ts

```
@@ -137,6 +137,11 @@ PR is no longer targeting this branch for a backport',
       let checkRun = allChecks.check_runs.find(run => run.name === VALID_BACKPORT_CHECK_NAME);
 
       if (pr.base.ref !== 'master') {
+        // If a branch is targeting something that isn't master it might not be a backport;
+        // allow for a label to skip backport validity check for these branches.
+        const hasSkipLabel = await labelExistsOnPR(context, pr.number, SKIP_CHECK_LABEL);
+        if (hasSkipLabel) return;
```

Will this dismiss the failure in the scenario that the label on the PR is initially forgotten, trop fails and then you later add the label to try and suppress the error? Would it potentially be better to update the check with an explicit success?

---

ah good point - i'll add some logic for labeling/unlabeling specifically.

---

looks great now 👍